packages/calling-component-bindings/src/handlers/createTeamsCallHandlers.ts

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { StartCallOptions } from '@azure/communication-calling';
+/* @conditional-compile-remove(teams-identity-support) */
+/* @conditional-compile-remove(PSTN-calls) */
+import { AddPhoneNumberOptions } from '@azure/communication-calling';
+/* @conditional-compile-remove(teams-identity-support) */
+import { TeamsCall, TeamsCallAgent } from '@azure/communication-calling';
+import { CommunicationIdentifier, isCommunicationUserIdentifier } from '@azure/communication-common';
+/* @conditional-compile-remove(teams-identity-support) */
+import { isPhoneNumberIdentifier } from '@azure/communication-common';
+import { Common, _toCommunicationIdentifier } from '@internal/acs-ui-common';
+import { StatefulCallClient, StatefulDeviceManager } from '@internal/calling-stateful-client';
+import memoizeOne from 'memoize-one';
+import { ReactElement } from 'react';
+import { isTeamsCallParticipants } from '../utils/callUtils';
+import { createDefaultCommonCallingHandlers, SharedCallingHandlers } from './createCommonHandlers';
+
+/**
+ * Object containing all the teams call handlers required for calling components.
+ *
+ * Calling related components from this package are able to pick out relevant handlers from this object.
+ * See {@link useHandlers} and {@link usePropsFor}.
+ *
+ * @beta
+ */
+export type TeamsCallingHandlers = SharedCallingHandlers & {
+  onStartCall: (
+    participants: CommunicationIdentifier[],
+    options?: StartCallOptions
+  ) => void | /* @conditional-compile-remove(teams-identity-support) */ TeamsCall;
+};
+
+/**
+ * Create the default implementation of {@link TeamsCallingHandlers} for teams call.
```

Comment:
> for teams call

These are handlers if you are using a teams identity. You could still have a teams interop call using the acs call objects right?

Reply:
Yep, teams interop is still an ACS call, but joining a teams link as a locator (people still pay ACS calling bill)
Teams Call is the one use AAD token directly (which they pay teams/license)